test/e2e/agent/agent_updates_test.go

```diff
@@ -64,7 +64,7 @@ var _ = Describe("VM Agent behavior during updates", func() {
 				func(device *v1alpha1.Device) bool {
 					return device.Status.Os.Image == newImageReference &&
 						conditionExists(device, "Updating", "False", string(v1alpha1.UpdateStateUpdated))
-				}, "2m")
+				}, "5m")
```

Comment:
note: Switching from the local container storage generally takes about a minute. On top of the time it takes to pull the os image, upgrade takes a bit longer than timeout.